addon/ng2/utilities/get-dependent-files.ts

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+import * as fs from 'fs';
+import * as ts from 'typescript';
+import * as glob from 'glob';
+import * as path from 'path';
+
+/**
+ * Declare a MapType where key is a string and value is an array of type string
+ * Use for storing a file's path as key and its dependent module's relative path
+ */
+declare interface ModuleImport {
```

Comment:
You don't need the `declare` keyword here - interfaces are automatically declarations.